### PR TITLE
Feature/css styling

### PR DESCRIPTION
**Pull request by Cyanne-Jones**
This branch added CSS styling to the entire page and now matches the project comp.

### CODE REVIEW

**Review by catlynbowles — APPROVED** [review]
Everything looks great, matches comp to a tee! Waiting for Victoria's feedback before merge.

**Review by VictoriaFC — APPROVED** [review]
I agree! Looks great. Catlyn - feel free to merge to main!

**Pull request merged**
